src/sentry/api/endpoints/organization_plugins_configs.py

```diff
@@ -34,11 +34,6 @@ def get(self, request, organization):
         if not desired_plugins:
             desired_plugins = list(plugins.plugin_that_can_be_configured())
 
-        # True if only one plugin is desired, and it has been shadow deprecated
-        is_shadow_deprecated_detailed_view = (
-            len(desired_plugins) == 1 and desired_plugins[0].slug in SHADOW_DEPRECATED_PLUGINS
-        )
-
```

Comment:
The check on L87 already solves this, I probably didn't notice when I wrote this.